library/core/src/task/wake.rs

```diff
@@ -250,7 +250,8 @@ pub struct Context<'a> {
 impl<'a> Context<'a> {
     /// Create a new `Context` from a [`&Waker`](Waker).
     #[stable(feature = "futures_api", since = "1.36.0")]
-    #[rustc_const_unstable(feature = "const_waker", issue = "102012")]
+    #[rustc_const_stable(feature = "const_waker", since = "CURRENT_RUSTC_VERSION")]
+    #[rustc_allow_const_fn_unstable(local_waker)]
```

Comment:
This can hide arbitrary features. Why wasn't the method made const stable instead? Unstable things can be const stable just fine